cmd/auctioneer/config/config_test.go

```diff
@@ -99,6 +100,7 @@ var _ = Describe("AuctioneerConfig", func() {
 			BBSClientSessionCacheSize: 100,
 			BBSMaxIdleConnsPerHost:    10,
 			CACertFile:                "/path-to-cert",
+			BinPackFirstFitWeight:     0.0,
```

Comment:
test a non-zero value to validate that it is passing it through and not using the default Go value for float64.